scripts/wish/reduce.py

```diff
@@ -1,919 +1,470 @@
-# flake8: noqa
-import sys
-from mantid.simpleapi import *
-import numpy as n
+from __future__ import (absolute_import, division, print_function)
+from mantid import logger
 import mantid.simpleapi as mantid
+import os
```

Comment:
I see that this was not your change, but import `mantid.simpleapi as mantid` is a very bad idea. What if you need something from mantid that it's not in the simpleapi?

Reply:
changed it to use simple instead, allowing mantid to be imported seperatly